app/views/layouts/rails_email_preview/application.html.erb

```diff
@@ -2,12 +2,12 @@
 <html>
 <head>
   <title><%= t('.head_title') %></title>
-  <%= stylesheet_link_tag 'rails_email_preview/application', 'data-turbolinks-track': 'reload' %>
   <%= csrf_meta_tag %>
   <%= csp_meta_tag if RailsEmailPreview.rails_supports_csp_nonce? %>
-  <%= javascript_include_tag 'rails_email_preview/application',
-                             'data-turbolinks-track': 'reload' %>
   <%= favicon_link_tag 'rails_email_preview/favicon.png' %>
+  <style>
+    <%= raw render(file: "#{RailsEmailPreview::Engine.root}/app/assets/stylesheets/application.css") %>
```

Comment:
this is def gross but since we are using the gem might as well keep the grossness here rather than in stellar